<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(38, 38, 38);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_header">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(29, 29, 29);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="header_center_frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4"/>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>41</width>
            <height>41</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>../img/icon.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>10</y>
            <width>111</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>My Player</string>
          </property>
         </widget>
        </widget>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QFrame" name="header_right_frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="minimize_window_button">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../python/sys_info/sys_info/icons/cil-minus.png</normaloff>../../python/sys_info/sys_info/icons/cil-minus.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="restore_window_button">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../python/sys_info/sys_info/icons/cil-window-restore.png</normaloff>../../python/sys_info/sys_info/icons/cil-window-restore.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="close_window_button">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../python/sys_info/sys_info/icons/X.png</normaloff>../../python/sys_info/sys_info/icons/X.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignTop">
     <widget class="QFrame" name="frame_top">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>320</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QFrame" name="frame_img">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_img">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>../img/Music.jpg</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_controls">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="btn_back">
               <property name="styleSheet">
                <string notr="true">QPushButton{
border-width: 5px;
border-style: solid;
border-radius: 25px;;
border-color: rgb(24, 24, 24);
background-color: qlineargradient(spread:pad, x1:0.474, y1:1, x2:0.484221, y2:0, stop:0.0328638 rgba(0, 0, 0, 255), stop:0.478873 rgba(56, 56, 56, 255), stop:0.957746 rgba(98, 98, 98, 255));
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0.46, y1:0, x2:0.488915, y2:1, stop:0.0328638 rgba(0, 0, 0, 255), stop:0.488263 rgba(56, 56, 56, 255), stop:0.957746 rgba(98, 98, 98, 255));
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../img/back.png</normaloff>../img/back.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_play">
               <property name="styleSheet">
                <string notr="true">QPushButton{
border-width: 5px;
border-style: solid;
border-radius: 30px;;
border-color: rgb(24, 24, 24);
background-color: qlineargradient(spread:pad, x1:0.474, y1:1, x2:0.484221, y2:0, stop:0.0328638 rgba(0, 0, 0, 255), stop:0.478873 rgba(56, 56, 56, 255), stop:0.957746 rgba(98, 98, 98, 255));
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0.46, y1:0, x2:0.488915, y2:1, stop:0.0328638 rgba(0, 0, 0, 255), stop:0.488263 rgba(56, 56, 56, 255), stop:0.957746 rgba(98, 98, 98, 255));
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../img/play.png</normaloff>../img/play.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>55</width>
                 <height>55</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_next">
               <property name="styleSheet">
                <string notr="true">QPushButton{
border-width: 5px;
border-style: solid;
border-radius: 25px;;
border-color: rgb(24, 24, 24);
background-color: qlineargradient(spread:pad, x1:0.474, y1:1, x2:0.484221, y2:0, stop:0.0328638 rgba(0, 0, 0, 255), stop:0.478873 rgba(56, 56, 56, 255), stop:0.957746 rgba(98, 98, 98, 255));
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0.46, y1:0, x2:0.488915, y2:1, stop:0.0328638 rgba(0, 0, 0, 255), stop:0.488263 rgba(56, 56, 56, 255), stop:0.957746 rgba(98, 98, 98, 255));
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../img/next.png</normaloff>../img/next.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QFrame" name="frame_4">
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QSlider" name="music_volume">
               <property name="styleSheet">
                <string notr="true">QSlider::groove:vertical {
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:1, stop:0 rgba(22, 22, 22, 255), stop:0.446009 rgba(40, 40, 40, 255), stop:0.699531 rgba(40, 40, 40, 255), stop:1 rgba(29, 29, 29, 255));
	border-radius: 5px;
    position: absolute; /* absolutely position 4px from the left and right of the widget. setting margins on the widget should work too... */
    left: 4px; right: 4px;
}

QSlider::handle:vertical {
    height: 15px;
	background-color: rgb(97, 97, 97);
    margin: 0 -4px; /* expand outside the groove */
	border-radius: 5px;
}

QSlider::add-page:vertical {
    border-radius: 5px;
	
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 rgba(0, 49, 145, 255), stop:0.399061 rgba(0, 49, 145, 255), stop:0.7277 rgba(0, 87, 255, 255), stop:1 rgba(0, 87, 255, 255));
}
</string>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_volum_img">
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap>../img/volume.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_title">
               <property name="maximumSize">
                <size>
                 <width>350</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                 <bold>true</bold>
                 <underline>false</underline>
                 <strikeout>false</strikeout>
                 <stylestrategy>PreferDefault</stylestrategy>
                </font>
               </property>
               <property name="acceptDrops">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(5, 186, 188);</string>
               </property>
               <property name="text">
                <string>Unknown Title</string>
               </property>
               <property name="scaledContents">
                <bool>false</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_albom">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(131, 131, 136);</string>
               </property>
               <property name="text">
                <string>Unknown Albom - Track 10</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_bottom">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QSlider" name="music_slider">
         <property name="styleSheet">
          <string notr="true">QSlider::groove:horizontal {
	border-radius: 7px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(22, 22, 22, 255), stop:0.446009 rgba(40, 40, 40, 255), stop:0.699531 rgba(40, 40, 40, 255), stop:1 rgba(29, 29, 29, 255));
    position: absolute; /* absolutely position 4px from the left and right of the widget. setting margins on the widget should work too... */
    left: 4px; right: 4px;
}


QSlider::sub-page:horizontal {
    border-radius: 7px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 85, 255, 255), stop:0.262911 rgba(0, 89, 255, 255), stop:0.57277 rgba(0, 57, 162, 255), stop:1 rgba(0, 47, 141, 255));
}</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>25:10 / 70:10</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
